docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -3658,12 +3658,10 @@ IS_IPV4( expression )
 * `_expression_`
 +
 specifies an expression that determines the values to include in the
-computation of the minimum. The _expression_ cannot contain an aggregate
-function or a subquery. The DISTINCT clause specifies that the MIN
-function operates on distinct values from the one-column table derived
-from the evaluation of _expression_. All nulls are eliminated before
-the function is applied to the set of values. If the result table is
-empty, IS_IPV4 returns NULL.
+validation of the IP address. The _expression_ cannot contain an aggregate
+function or a subquery. All nulls are eliminated before the function is
+applied to the set of values. If the result table is empty, IS_IPV4 

```

Comment:
The last two sentences appear to be discussing sets or tables of values, but IS_IPV4 just takes a single value. Consider replacing "All nulls... returns NULL" with "If the input value is NULL, IS_IPV4 returns NULL." Incidentally, just for fun I tried doing select is_ipv4(max(b)) from temp1, where temp1 is a table containing a char column b, and I got an assertion failure.